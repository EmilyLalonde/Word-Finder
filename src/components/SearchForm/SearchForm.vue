<template>
  <div id="searchForm">
    <input />
    <button>Submit</button>
</div>
</template>

<script>
export default {
  
}
</script>

<style scoped>

</style>