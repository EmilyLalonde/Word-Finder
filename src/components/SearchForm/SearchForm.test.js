import { mount } from '@vue/test-utils';
import SearchForm from './SearchForm.vue';

describe('SearchForm', () => {

  test('renders correctly', () => {
    const wrapper = mount(SearchForm);
    expect(wrapper.element).toMatchSnapshot();
  })
})