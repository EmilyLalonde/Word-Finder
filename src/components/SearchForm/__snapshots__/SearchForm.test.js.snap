// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm renders correctly 1`] = `
<div
  id="searchForm"
>
  <input
    class=""
    placeholder="Search ..."
  />
   
  <button>
    Submit
  </button>
   
  <!---->
</div>
`;
